staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -453,6 +474,25 @@ func validateCustomResourceDefinitionSpec(ctx context.Context, spec *apiextensio
 		}
 	}
 
+	if len(spec.SelectableFields) > 0 {
+		// Only validate field paths if the schema is present.
```

Comment:
old data doesn't have to have structural schemas.  let's explicitly reject if there isn't a structural schema so the new feature can only be used when schemas are present.